pages/seeker/index.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import styles from './seeker.module.css'
+
+const lerp = (start: number, end: number, percent: number) => {
+   return start + (start - end) * percent
+}
+
+interface IFollowerProps {
+  children: React.ReactNode
+  followerRef: React.RefObject<HTMLDivElement>
+}
+
+const MouseFollower = (props: IFollowerProps) => {
+  const [followerLeft, setFollowerLeft] = React.useState(0)
+  const [followerTop, setFollowerTop] = React.useState(0)
+  const [mouseLeft, setMouseLeft] = React.useState(0)
+  const [mouseTop, setMouseTop] = React.useState(0)
+
+  React.useEffect(() => {
+    document.onmousemove = function(e) {
```

Comment:
You should use `addEventListener` here, and make sure to remove it in the exit callback.